runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -337,6 +339,33 @@ void openGroupConfigPanel(PluginListItem listItem, Config config, ConfigDescript
 				item.add(checkbox, BorderLayout.EAST);
 			}
 
+			if (cid.getType() == Range.class)
+			{
+				Range range = configManager.getConfiguration(cd.getGroup().value(), cid.getItem().keyName(), Range.class);
+
+				if (range == null)
+				{
+					try
+					{
+						range = (Range) ConfigInvocationHandler.callDefaultMethod(config, cid.getMethod(), null);
+					}
+					catch (Throwable throwable)
+					{
```

Comment:
Maybe you should do `continue` here? As something must be broken when this happens